frontend/common/src/lang/fr.json

```diff
@@ -1093,5 +1093,267 @@
   "BjRuMw": {
     "defaultMessage": "Voir la définition",
     "description": "Text displayed when skill block is open."
+  },
+  "+6TuVe": {
+    "defaultMessage": "Désélectionner {label}",
+    "description": "Text to uncheck checkbox button"
+  },
+  "/SBJ7g": {
+    "defaultMessage": "Sélectionnez {label}",
+    "description": "Text to check checkbox button"
+  },
+  "2Ka1oj": {
+    "defaultMessage": "Secret",
```

Comment:
The English string equivalement for this is _Top Secret_ which is not _Secret_ in French but _Très secret_. 

See "Niveau de la (des) vérification(s) de la fiabilité/sécurité requis" at https://www.tbs-sct.canada.ca/tbsf-fsct/330-23-html-fra.asp

Reply:
Thanks for flagging this!  Gray agrees with you.